swarm/storage/mru/update.go

```diff
@@ -17,304 +17,128 @@
 package mru
 
 import (
-	"bytes"
 	"encoding/binary"
 	"errors"
-	"fmt"
-	"hash"
 
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/swarm/log"
 	"github.com/ethereum/go-ethereum/swarm/multihash"
 	"github.com/ethereum/go-ethereum/swarm/storage"
 )
 
-// Signature is an alias for a static byte array with the size of a signature
-const signatureLength = 65
-
-type Signature [signatureLength]byte
-
-// SignedResourceUpdate contains signature information about a resource update
-type SignedResourceUpdate struct {
-	resourceUpdate // actual content that will be put on the chunk, less signature
-	signature      *Signature
-	updateAddr     storage.Address // resulting chunk address for the update
+// resourceUpdate encapsulates the information sent as part of a resource update
+type resourceUpdate struct {
+	updateHeader        // metainformationa about this resource update
+	data         []byte // actual data payload
 }
 
 // Update chunk layout
-// Header:
-// 2 bytes header Length
+// Prefix:
+// 2 bytes updateHeaderLength
 // 2 bytes data length
-// 4 bytes period
-// 4 bytes version
-// 32 bytes rootAddr reference
-// 32 bytes metaHash digest
+const prefixLength = 2 + 2
```

Comment:
Please name so we know what data it prefixes.

Reply:
ok. fixed.